### ISSUE TITLE
Support 1Password CLI 2 with biometric authentication

### ISSUE THREAD

**Issue by srijan** [fmartingr/qute-1password#8]
Ref: https://developer.1password.com/docs/cli/upgrade

Biometric unlock removes the need for explicit login and keeping track of session tokens.

**Comment by fmartingr** [comment]
I belive this is enabled directly on 1password:

![Screen Shot 2022-05-16 at 12 01 30 AM](https://user-images.githubusercontent.com/812088/168495673-55430fc1-29a4-4020-ae9e-7f6fe8775293.png)

So the only thing this script would need is a flag that disables the password prompt, am I right?

**Comment by srijan** [comment]
No, in 1password CLI 2, the command structure has changed and it's not backward compatible: https://developer.1password.com/docs/cli/upgrade/#step-2-update-your-scripts

For example, `op get item` will no longer work. It's been replaced by `op item get`.
